### ISSUE TITLE
libtdjni Java compilation error:  Ok cannot be converted to Error

### ISSUE THREAD

**Issue by eugeneRover** [tdlib/td#2660]
[This commit](https://github.com/tdlib/td/commit/dd77e4628f1a65f332f0cb81c82a19c0fcfa40c2) breaks java Example compilation.
The [following code](https://github.com/tdlib/td/blob/master/example/java/org/drinkless/tdlib/example/Example.java#L306C1-L308C10) :
```
        if (Client.execute(new TdApi.SetLogStream(new TdApi.LogStreamFile("tdlib.log", 1 << 27, false))) instanceof TdApi.Error) {
            throw new IOError(new IOException("Write access to the current directory is required"));
        }
```
now fails to compile (`cmake --build . --target install`) with an error:
```
[ 60%] Building Java code
/home/user/my-workspace/td/example/java/org/drinkless/tdlib/example/Example.java:306: error: incompatible types: Ok cannot be converted to Error
        if (Client.execute(new TdApi.SetLogStream(new TdApi.LogStreamFile("tdlib.log", 1 << 27, false))) instanceof TdApi.Error) {
                          ^
Note: /home/user/my-workspace/td/example/java/org/drinkless/tdlib/Client.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
gmake[2]: *** [CMakeFiles/build_java.dir/build.make:77: CMakeFiles/build_java] Ошибка 1
gmake[1]: *** [CMakeFiles/Makefile2:154: CMakeFiles/build_java.dir/all] Ошибка 2
gmake: *** [Makefile:149: all] Ошибка 2
```

**Comment by levlam** [comment]
Yes. The regression was fixed in TDLib 1.8.21, which will be released soon. For now, I would recommend to revert the last master commit and build the example without it.

**Comment by eugeneRover** [comment]
Fixed with release 1.8.21, compilation now succeeds
